src/widgets/ot_color_picker.eliom

```diff
@@ -148,6 +148,15 @@ let display ?a cp_sig =
   >|= [%shared display_aux ?a:~%a ~setter:~%setter]
   |> Eliom_content.Html.R.node
 
-let make ?a =
-  let cp_sig = Eliom_shared.React.S.create (255, 1.0, 0.0) in
-  (display ?a cp_sig, fst cp_sig)
+let make ?a ?update () =
+  let ((cp_get, cp_set) as cp_sig) =
+    Eliom_shared.React.S.create (255, 1.0, 0.0)
+  in
+  let (_ : unit Eliom_client_value.t) =
+    [%client
+      match ~%update with
+      | None -> ()
+      | Some update ->
+          ignore (React.E.map (fun update -> ~%cp_set update) update)]
```

Comment:
Or may be use an event `(cp_ev, ep_send)` instead of `(cp_sig, c_set)`, and make the function return something like `E.hold (E.select [cp_ev; update])`